### ISSUE TITLE
Copy HTML

### ISSUE THREAD

**Issue by viniamadeu** [TBag/power-apps-copy-text-to-clipboard#22]
Hello, 

how is it possible to copy HTML content instead of text?

Congratulations on the initiative, you are helping a lot of people

**Comment by planetparker** [comment]
Yes, it is possible.  This code relies upon the copy-to-clipboard library.  You'll find that the copy() function accepts a parameter which allows you to specify the format.

**In this case, if you change Todd's code in the index.ts file from this:**
copy(this._value.toString())

**to this:**
copy(this._value.toString(), {format: 'text/html'})

That will copy as html.  I am working on a branch that will include this functionality, but unsure when I'll get it posted.
